components/frontend-react/src/containers/Dashboard.js

```diff
@@ -61,27 +63,25 @@ const styles = theme => ({
 
 class Dashboard extends React.Component {
 
-  constructor(){
-    super();
+  constructor(props){
+    super(props);
     this.state = {
+      link: '',
       completedList:[],
       queuedList:[]
     }
-  }
-
-  componentWillMount() {
     let token = sessionStorage.getItem('token');
     axios({
-      method:'get',
-      url: `${process.env.REACT_APP_API_URL}/api/user/downloads/1`,
+      method: 'get',
+      url: `${process.env.REACT_APP_API_GET_DOWNLOADS}`,
       headers: {'token': `${token}`}
     })
-    .then( res => {
-      let completedList = res.data.filter(file => file.status === 3);
+    .then(res => {
       let queuedList = res.data.filter(file => file.status === 0);
-      this.setState({completedList, queuedList})
+      let completedList = res.data.filter(file => file.status === 3);
```

Comment:
let's not have magic numbers, define these number as constants somewhere and then use. 
```
const PENDING = 2;
const DOWNLOADING = 3;

...

.then(res => {
   let completedList = res.data.filter(file => file.status === DOWNLOADING);
``